diff --git a/Makefile b/Makefile
index d652cdf..d01ebf1 100644
--- a/Makefile
+++ b/Makefile
@@ -17,7 +17,7 @@ LMATH= liblapack.a libblas.a
 #
 OBJS=init_once.o init_dyn21.o dynm.o banner.o sec.o dyn21.o dyn21umat.o dyn21umats.o dyn21umatv.o dyn21umatc.o dyn21tumat.o dyn21utan.o dyn21ueos.o dyn21cnt.o dyn21ushl.o dyn21usld.o  couple2other_user.o dynrfn_user.o adummy_graph.o orderByMetis.o mod_impl_matrices.o 
 #
-LIBS=libdyna.a liblsmpp.a libmpp_lanczos.a libbcsext4.a liblsda.a liblsdb.a liblssecurity.a  liblcpack.a libmf2.a libspooles.a liblcpack.a libspooles.a libcparse.a liblsm.a liblscrypt.a libresurf.a liblsmppdes.a libdsa.a liblsmc.a libbalance.a libsfg.a       libmetis.a     liblsmpp.a liblso.a libunits.a libfftw3.a     libdyna.a libmetis.a libspooles.a
+LIBS=libdyna.a liblsmpp.a libmpp_lanczos.a libbcsext4.a liblsda.a liblsdb.a liblssecurity.a  liblcpack.a libmf2.a libspooles.a liblcpack.a libspooles.a libcparse.a liblsm.a liblscrypt.a libresurf.a liblsmppdes.a libdsa.a liblsmc.a libbalance.a libsfg.a       libmetis.a     liblsmpp.a liblso.a libunits.a libfftw3.a libansyslic.a libdyna.a libmetis.a libspooles.a
 #
 mppdyna: $(OBJS)
 	$(LD) -o mppdyna $(OBJS) $(LIBS) $(LMATH) $(LF)
