diff --git a/Makefile b/Makefile
index 6434685..aeeb446 100644
--- a/Makefile
+++ b/Makefile
@@ -10,19 +10,19 @@
 # below. Example:
 # LMATH=
 #
-FC=/opt/intel/compilers_and_libraries_2019/linux/bin/intel64/ifort -qopenmp
+FC=ifort -qopenmp
 #
 FF=-O2 -safe_cray_ptr -assume byterecl,buffered_io,protect_parens -warn nousage -zero -ftz -fp-model precise -mP2OPT_hpo_dist_factor=21 -diag-disable 10212,10010 -pad -qopenmp -i8 -r8 -fimf-arch-consistency=true -qno-opt-dynamic-align -xSSE2 -align array16byte -fPIC
 #
-LD=/opt/intel/compilers_and_libraries_2019/linux/bin/intel64/ifort -qopenmp -sox
+LD=ifort -qopenmp -sox
 #
 LF= -qopenmp-link=static -static-intel  -qopenmp -lrt -lstdc++ -lpthread -lstdc++ -lpthread
 #
-LMATH= liblapack.a libblas.a
+LMATH= -Wl,--start-group ${MKLROOT}/lib/intel64/libmkl_intel_ilp64.a ${MKLROOT}/lib/intel64/libmkl_intel_thread.a ${MKLROOT}/lib/intel64/libmkl_core.a -Wl,--end-group
 #
 OBJS=init_once.o init_dyn21.o dynm.o banner.o sec.o dyn21.o dyn21umat.o dyn21umats.o dyn21umatv.o dyn21umatc.o dyn21tumat.o dyn21utan.o dyn21ueos.o dyn21cnt.o dyn21ushl.o dyn21usld.o  dyn21em.o dyn21icfd.o couple2other_user.o dynrfn_user.o adummy_graph.o orderByMetis.o mod_impl_matrices.o 
 #
-LIBS=libdyna.a libbcsext4.a liblsda.a liblsdb.a liblssecurity.a  liblcpack.a libmf2.a libspooles.a liblcpack.a libspooles.a libcparse.a liblsm.a liblscrypt.a libresurf.a libdsa.a liblsmc.a libsfg.a libevl.a  libcminpack.a  libzmumps.a libdmumps.a libmumps_common.a libmumps_mpiseq.a    libmetis.a libim_rotor_dynamics.a libarpack.a  libmcms.a   liblso.a libunits.a  libem.a libfemster_wrap.a libfemster_wrap2d.a libfemster_wrap1d.a libfemster.a libfemster2d.a libfemster1d.a libpfem.a libcese.a libchemistry.a libem_tailsit.a libparticles.a libeosfl2.a  libpfem.a libmoving.a libdualcese.a libbattery.a libfftw3.a libfftw3_omp.a     libdyna.a libmetis.a libspooles.a
+LIBS=libdyna.a libbcsext4.a liblsda.a liblsdb.a liblssecurity.a  liblcpack.a libmf2.a libspooles.a liblcpack.a libspooles.a libcparse.a liblsm.a liblscrypt.a libresurf.a libdsa.a liblsmc.a libsfg.a libevl.a  libcminpack.a  libzmumps.a libdmumps.a libmumps_common.a libmumps_mpiseq.a    libmetis.a libim_rotor_dynamics.a libarpack.a  libmcms.a   liblso.a libunits.a  libem.a libfemster_wrap.a libfemster_wrap2d.a libfemster_wrap1d.a libfemster.a libfemster2d.a libfemster1d.a libpfem.a libcese.a libchemistry.a libem_tailsit.a libparticles.a libeosfl2.a  libpfem.a libmoving.a libdualcese.a libbattery.a libfftw3.a libfftw3_omp.a libansyslic.a libdyna.a libmetis.a libspooles.a
 #
 lsdyna: $(OBJS)
 	$(LD) -o lsdyna $(OBJS) $(LIBS) $(LMATH) $(LF)
